### PR TITLE
Upgrade packages and freeze current docs

### PR DESCRIPTION
**Pull request by hans-notifi**
- Upgrade all dependent packages (e.g. React, Docusaurus) to their latest versions
- Save the current version of the docs as version 1.1.0, in preparation for upcoming overhaul

### CODE REVIEW

File: versioned_docs/version-1.1.0/advanced/xmtp-source.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
+# Alerts for new XMTP messages
+
+This page will walk through an integration of XMTP as a [source](../integration-overview/alerts-in-depth#source) using the **Notifi React Card** and **Admin Portal**. This will alert an user whenever they receive a new message to any XMTP conversation they have at the time of subscribing.
```
Comment (bruno-notifi) [inline versioned_docs/version-1.1.0/advanced/xmtp-source.md:3]: What's Notifi React Card?

Comment (hans-notifi) [reply versioned_docs/version-1.1.0/advanced/xmtp-source.md:3]: Our easy plug-and-play solution for creating a Notifi subscription UI.

This is the `version-1.1.0` folder, though. All of the actual docs fixes will go in the `docs` folder.

File: versioned_docs/version-1.1.0/alert-design/custom-alert-templates.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
+# Custom Alert Templates
+
+<!-- https://docs.google.com/document/d/1-8K_nbJGaywL-jYFg7_YZiE9x_WPaHvuFfYgM7QmqMg/edit -->
+
+Notifi uses templates to render the same notification in different forms for different channels such as email, Telegram, and SMS. We use [Mustache templates](https://mustache.github.io/mustache.5.html), and you can try out a demo here: https://mustache.github.io/#demo
```
Comment (gustavo-notifi) [inline versioned_docs/version-1.1.0/alert-design/custom-alert-templates.md:5]: the problem with this is that we're expecting {{{ }}} in a few templates... idk which, but I know a few of mine need triple while others needs doubles

**Review by gustavo-notifi — APPROVED** [review]
maybe I missed it, but are we mentioning alert frequencies as well?

I think something we should let the admin know in the UI as well is how it works if they change the frequency from an old alert, since it won't affect users that have been subscribed before the change...

File: versioned_docs/version-1.1.0/alert-design/custom-alert-templates.md (hunk 2 above)
Comment (hans-notifi) [reply versioned_docs/version-1.1.0/alert-design/custom-alert-templates.md:5]: Yeah, I can address this in the new docs

**Pull request merged**